### DISCUSSION TITLE
Render-Only library?

### DISCUSSION THREAD

**Discussion by IceReaper** [o3de/o3de#1975] [Q&A]
While working on my prototype for years I'm always lacking one thing: a pure renderer library with bindings for multiple languages (c# in my case). Are there any plans to split this project into multiple smaller libraries (and having bindings for various languages ofcourse) to let people pick the features they want to use using their own app workflow and setup? Monogame and libGDX both try to archive that somewhat, but they definitely lack any good state of the art renderer.

**Comment by rgba16f** [comment]
One of the longer term plans for O3DE is to break it down into separate libraries. Atom would be one of those libraries and we want to enable people to use it in products outside O3DE. That said it will rely on the core AzCore & AzFramework O3DE libraries.

Atom bindings for languages other than c++ is not something that has been explored yet.
